src/index.html

```diff
@@ -2,13 +2,103 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="menu">
+        <ul class="menu__item">
+          <li class="menu__items">
```

Comment:
Please fix it everywhere 
```suggestion
          <li class="menu__item">
```